api/src/iot/import_utils.py

```diff
@@ -79,6 +83,33 @@ def get_center_coordinates(postcode: str, house_number: Union[int, str]) -> Poin
     raise PostcodeSearchException(postcode, house_number)
 
 
+class InvalidPersonDataError(ValidationError):
+    def __str__(self):
+        fields = self.args[0].detail.serializer.fields
+        return "<br>".join(
+            fields[field].source + ': ' + (','.join(str(e) for e in error_detail))
+            for field, error_detail in self.args[0].args[0].items()
+        )
+
+
+class RequiredCharField(fields.CharField):
+    def __init__(self, **kwargs):
+        kwargs.setdefault('allow_blank', False)
+        kwargs.setdefault('allow_null', False)
+        super().__init__(**kwargs)
+
+
+class PersonDataSerializer(Serializer):
+    organisation = RequiredCharField(max_length=255, source="Naam organisatie/bedrijf")
```

Comment:
Wat als ik als privé-entiteit (los van een bedrijf) mijn sensor meld? Dan zou dit veld leeg moeten kunnen zijn, denk ik.

Reply:
Zoals het formulier nu werkt is organisatie een verplicht veld. Er valt inderdaad wat voor te zeggen om dat optioneel te maken. Ik zal het in ieder geval hier zo maken dat een lege waarde importeerbaar is.